recipes-bsp/atf/atf-poplar_git.bb

```diff
@@ -0,0 +1,41 @@
+DESCRIPTION = "ARM Trusted Firmware Poplar"
```

Comment:
Yes, we could add it into U-Boot recipe. However I kept it separate as it seems to me that will create ATF duplication when you are supporting both U-Boot and UEFI in BL33. For example adding tee.bin, you will end up  having to update both uefi & u-boot recipes each time you need to change some ATF configuration. I guess we haven't hit that problem with HiKey yet as we never added support for U-Boot (yet?) in OE builds.

Or maybe it should be a atf.inc, and then included form both recipes..